### DISCUSSION TITLE
How to proceed in order to have multilingual site?

### DISCUSSION THREAD

**Discussion by ericpar** [reuixiy/hugo-theme-meme#297] [Q&A] [answered]
Hello,

I am wondering how to benefit from the multilingual capabilities with this theme?
Is there any example somewhere I could inspire myself from?

Regards,

Eric

**Comment by reuixiy** [comment, marked as answer]
Yes, check the examples below:

- https://github.com/reuixiy/io-oi.me/blob/5e2151b3729565f89905063150081d74da500cf3/config.toml#L1406-L1459
- https://github.com/Syralist/blog/blob/7d0385b352d5c9827e70295564b12919cf91ac9b/config.toml#L1366-L1381
